### PR TITLE
7107: PII flat set true in dev

### PR DESCRIPTION
**Pull request by RuthBerry96**
Story: [AB#7103](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7103)
Task [AB#7107](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7107)

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs [hunk 1]
```diff
@@ -21,11 +22,14 @@ namespace NHSD.BuyingCatalogue.Organisations.Api
 {
     public class Startup
     {
+        private readonly IWebHostEnvironment _enviroment;
+
         private const string BearerToken = "Bearer";
 
-        public Startup(IConfiguration configuration)
+        public Startup(IConfiguration configuration, IWebHostEnvironment enviroment)
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs:29]: Typo: `enviroment` should be `environment`.

File: src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs [hunk 2]
```diff
@@ -21,11 +22,14 @@ namespace NHSD.BuyingCatalogue.Organisations.Api
 {
     public class Startup
     {
+        private readonly IWebHostEnvironment _enviroment;
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs:25]: Typo: `_enviroment` should be `_environment`.

Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs:25]: 👍

File: src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs (hunk 1 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs]: 👍

**Pull request merged**